java/src/main/java/com/logicalclocks/hsfs/engine/Utils.java

```diff
@@ -141,7 +141,8 @@ public String getFgName(FeatureGroup featureGroup) {
   }
 
   public String getHiveMetastoreConnector(FeatureGroup featureGroup) throws IOException, FeatureStoreException {
```

Comment:
This method name is wrong. The connection is not for the metastore, but for the hiveserver

Reply:
fix by applying #344